src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -535,6 +535,10 @@ public OptionalArrayMethodView generateDynamicLangApiMethod(MethodTransformerCon
         generateRequestObjectParams(context, filteredFieldConfigs));
     apiMethod.grpcStreamingType(context.getMethodConfig().getGrpcStreamingType());
 
+    ServiceMessages messages = new ServiceMessages();
+    apiMethod.isLongRunning(
+        messages.isLongRunningOperationType(context.getMethod().getOutputType()));
+
```

Comment:
This can be simplified as 

apiMethod.isLongRunning(context.isLongRunning(context.getMethod()));